authorized_keys_command.sh

```diff
@@ -10,6 +10,26 @@ SaveUserName=${SaveUserName//"="/".equal."}
 SaveUserName=${SaveUserName//","/".comma."}
 SaveUserName=${SaveUserName//"@"/".at."}
 
-aws iam list-ssh-public-keys --user-name "$SaveUserName" --query "SSHPublicKeys[?Status == 'Active'].[SSHPublicKeyId]" --output text | while read KeyId; do
-  aws iam get-ssh-public-key --user-name "$SaveUserName" --ssh-public-key-id "$KeyId" --encoding SSH --query "SSHPublicKey.SSHPublicKeyBody" --output text
+# Specify IAM group(s) separated by spaces to import users.
+# Specify "##ALL##" (including the double quotes) to import all users
+IAMGroup=("##ALL##")
```

Comment:
Personally I would do this in import_users.sh instead of here. Putting it here makes logging in with ssh even slower. And it is already slow because of the API calls.

While it works, this script does a lot now that it doesn't have to do. If you only import the users in the IAM groups, you also limit the amount of users on the system.

So I would suggest moving this code to the import_users.sh. The code itself looks good to me :)